apis/conversion/src/__tests__/index.spec.js

```diff
@@ -54,24 +57,32 @@ describe('objectConversion', () => {
     });
 
     it('should use hypercube.exportProperties and hypercube.importProperties when there is no source exportProperties and no target importProperties', async () => {
```

Comment:
This statement:
It should check the hyperCubePath as well. (Maybe it does already).

Reply:
Yes, I did check the hypercubePath.